ansible/roles/ecryptfs/tasks/main.yml

```diff
@@ -39,9 +39,9 @@
 
 - name: Create entry in fstab
   sudo: yes
-  lineinfile: 
+  lineinfile:
     dest: "/etc/fstab"
     regexp: "^/opt/data"
     backup: yes
     line: "{{ mount_configuration.stdout }}"
-  when: "'/opt/data' not in fstab_contents.stdout"
+  when: "'/opt/data' not in mtab_contents.stdout"
```

Comment:
I think so, the regexp finds the line to replace (if there is a change), but there is no need to replace that line when we are not mounting the drive